keras/layers/normalization/spectral_normalization.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2023 The Keras Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import tensorflow.compat.v2 as tf
+
+from keras.layers.rnn import Wrapper
+
+# isort: off
+from tensorflow.python.util.tf_export import keras_export
+
+
+# Adapted from TF-Addons implementation
+@keras_export("keras.layers.SpectralNormalization", v1=[])
+class SpectralNormalization(Wrapper):
+    """Performs spectral normalization on weights.
```

Comment:
Please format the docstring by adding line breaks everywhere needed.

Reply:
Ok.